python/tk_houdini_alembicnode/handler.py

```diff
@@ -0,0 +1,521 @@
+            # of rendered frames instead:
+            rendered_files = self._get_rendered_files(current_node)
+
+            if not rendered_files:
+                msg = ("Unable to find rendered files for node '%s'." 
+                       % (current_node,))
+                self._app.log_error(msg)
+                hou.ui.displayMessage(msg)
+                return
+            else:
+                render_dir = os.path.dirname(rendered_files[0])
+
+        # if we have a valid render path then show it:
+        if render_dir:
+            # TODO: move to utility method in core
+            system = sys.platform
+
+            # run the app
+            if system == "linux2":
+                cmd = "xdg-open \"%s\"" % render_dir
+            elif system == "darwin":
+                cmd = "open '%s'" % render_dir
+            elif system == "win32":
+                cmd = "cmd.exe /C start \"Folder\" \"%s\"" % render_dir
+            else:
+                msg = "Platform '%s' is not supported." % (system,)
+                self._app.log_error(msg)
+                hou.ui.displayMessage(msg)
+
+            self._app.log_debug("Executing command:\n '%s'" % (cmd,))
+            exit_code = os.system(cmd)
+            if exit_code != 0:
+                msg = "Failed to launch '%s'!" % (cmd,)
+                hou.ui.displayMessage(msg)
+
+    # lookup the default node name from the settings and apply it to the
+    # supplied node
+    def set_default_node_name(self, node):
+        default_name = self._app.get_setting('default_node_name')
+        return node.setName(default_name, unique_name=True)
+
+
+    ############################################################################
+    # Private methods
+
+
+    # compute the output path based on the current work file and cache template
+    def _compute_output_path(self, node):
+
+        # get relevant fields from the current file path
+        work_file_fields = self._get_hipfile_fields()
+
+        if not work_file_fields:
+            msg = "This Houdini file is not a Shotgun Toolkit work file!"
+            raise sgtk.TankError(msg)
+
+        # Get the cache templates from the app
+        work_cache_template = self._app.get_template("work_cache_template")
+
+        # create fields dict with all the metadata
+        fields = {
+            "name": work_file_fields.get("name", None),
+            "node": node.name(),
+            "renderpass": node.name(),
+            "SEQ": "FORMAT: $F",
+            "version": work_file_fields.get("version", None),
+        }
+
+        # get the camera width and height if necessary
+        if ("width" in work_cache_template.keys or 
+            "height" in work_cache_template.keys):
+            cam_path = node.parm("geometry1_camera").eval()
+            cam_node = hou.node(cam_path)
+            if not cam_node:
+                raise sgtk.TankError("Camera %s not found." % cam_path)
```

Comment:
I'm confused by this... for an alembic cache, why would the output be dependent on the resolution of the camera (if there even is such a param).  Is this just something left in from a copy/paste from another node?